src/lib.rs

```diff
@@ -0,0 +1,62 @@
+pub mod domain;
+pub mod pubkey;
+pub mod keypair;
+pub mod roinput;
+pub mod signature;
+pub mod schnorr;
+
+pub use domain::*;
+
+pub use pubkey::PubKey;
+pub use pubkey::PubKeyHelpers;
+pub use pubkey::CompressedPubKey;
+pub use keypair::Keypair;
+pub use roinput::{
+    Input,
+    ROInput,
+};
+pub use signature::Signature;
+pub use schnorr::Schnorr;
+
+use oracle::{
+    pasta,
+    poseidon::{
+        Sponge,
+        ArithmeticSponge,
+        SpongeConstants,
+        ArithmeticSpongeParams,
+        PlonkSpongeConstants,
+    },
+};
+
+#[derive(Copy, Clone)]
+#[repr(u8)]
```

Comment:
why `repr(u8)`?

Reply:
I think rust-analyser put that there.  I'll try to remove and see if it puts it back.

Reply:
Ah, I remember, I want to make sure that it's represented as a byte and I want to explicitly state this. It's part of the public facing specification.

Reply:
Fixed